ibm/service/power/resource_ibm_pi_dhcp.go

```diff
@@ -65,6 +65,13 @@ func ResourceIBMPIDhcp() *schema.Resource {
 				Description: "Optional name of DHCP Service (will be prefixed by DHCP identifier)",
 				ForceNew:    true,
 			},
+			Arg_DhcpSnatEnabled: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Indicates if SNAT will be enabled for the DHCP service",
+				ForceNew:    true,
+			},
```

Comment:
Can we set d.Set in read method

Reply:
At this time the service broker does not provide a SNAT enabled value on the DHCP server read endpoint. This was done by design per the service broker developer.